packages/mephisto-worker-addons/src/index.jsx

```diff
@@ -1,3 +1,3 @@
 import Tips from "./Tips/index";
-import { createFeedback, createTip } from "./Functions";
-export { Tips, createFeedback, createTip };
+import Feedback from "./Feedback/index";
+export { Tips, Feedback };
```

Comment:
The createFeedback & createTip methods are way more involved/complex than they previously were, so it makes not sense for them to be exported for the user to run.